bibliopixel/drivers/serial/devices.py

```diff
@@ -41,6 +49,13 @@ def find_serial_devices(self):
         return self.devices
 
     def get_device(self, id=None):
+        """Returns details of either the first or specified device
```

Comment:
You explain the "first or" twice - no need.

    Returns details for an individual device